[2022-02-03 18:59:04,652] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-01T09:00:00+00:00 [queued]>
[2022-02-03 18:59:04,658] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-01T09:00:00+00:00 [queued]>
[2022-02-03 18:59:04,658] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:59:04,658] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-02-03 18:59:04,658] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:59:04,665] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-02-01T09:00:00+00:00
[2022-02-03 18:59:04,676] {standard_task_runner.py:54} INFO - Started process 10421 to run task
[2022-02-03 18:59:04,739] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-02-01T09:00:00+00:00', '--job_id', '15', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpq635xs_x']
[2022-02-03 18:59:04,740] {standard_task_runner.py:78} INFO - Job 15: Subtask twitter_aluraonline
[2022-02-03 18:59:04,776] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-02-01T09:00:00+00:00 [running]> on host SPVALN1-612R9D4.pitagoras.apollo.br
[2022-02-03 18:59:04,831] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-02-03 18:59:04,834] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-01T09:00:00.00Z&end_time=2022-02-02T09:00:00.00Z
[2022-02-03 18:59:09,098] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:59:09,477] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-01T09:00:00.00Z&end_time=2022-02-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hjoqscfayrhl2qbcpmrgkiikd
[2022-02-03 18:59:09,478] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:59:09,865] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-01T09:00:00.00Z&end_time=2022-02-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hhkwys6l0rqwxf4dhyjlgnvr1
[2022-02-03 18:59:09,866] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:59:10,165] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-02-01T09:00:00.00Z&end_time=2022-02-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe48hhjeogm8sr9iczajgyde47m9p
[2022-02-03 18:59:10,166] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:59:10,437] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220201T090000, start_date=20220203T215904, end_date=20220203T215910
[2022-02-03 18:59:14,643] {local_task_job.py:102} INFO - Task exited with return code 0
